recipes/dd-agent.rb

```diff
@@ -53,6 +53,11 @@
 #
 raise "Add a ['datadog']['api_key'] attribute to configure this node's Datadog Agent." if node['datadog'] && node['datadog']['api_key'].nil?
 
+api_keys = node['datadog']['api_key']
+dd_urls = node['datadog']['dd_url']
```

Comment:
this node attribute was previously named `node['datadog']['url']`:
- is the change intentional?
- if so are we sure that it's something that's only used by us internally? (I think so, but just to be extra sure)